Server/upload/95a14afff5747afd941776100c754b0d

```diff
@@ -0,0 +1,240 @@
+{
+  "reference": {
```

Comment:
remove the server files that were generated in the process but one that GD added, ill send you the id of it

Reply:
all but this: 
![image](https://user-images.githubusercontent.com/22376976/106404671-77a55180-6401-11eb-950c-f1c9a2f45922.png)